modules/programs/gh.nix

```
@@ -106,5 +115,12 @@ in {
     programs.git.extraConfig.credential."https://github.com".helper =
       mkIf cfg.enableGitCredentialHelper
       "${cfg.package}/bin/gh auth git-credential";
+
+    xdg.dataFile."gh/extensions" = mkIf (cfg.extensions != null) {
```

Perhaps it would suffice to have

```suggestion
    xdg.dataFile."gh/extensions" = mkIf (cfg.extensions != []) {
```

Then the `nullOr` could be removed.

---

I am not necessarily opposed, but did you see my remark from the PR description:

> I made the option nullable and null by default to avoid errors when people (probably on non-NixOS) used `gh extension install` before and hence already have a non-empty folder `.local/share/gh/extensions`.

WDYT?

---

Good point, will change :+1: